packages/react-vapor/src/components/table-hoc/tests/TableHOC.spec.tsx

```diff
@@ -90,10 +90,10 @@ describe('TableHOC', () => {
             expect(wrapper.find(ActionBarConnected).prop('extraContainerClasses')).toContain('mod-border-top');
         });
 
-        it('should render an ActionBarConnected with a top border if the "showBorderBottom" is set to true', () => {
-            const wrapper = shallow(<TableHOC {...defaultProps} hasActionButtons showBorderBottom />);
+        it('should render an ActionBarConnected without a top border if the "showBorderBottom" is set to false', () => {
+            const wrapper = shallow(<TableHOC {...defaultProps} hasActionButtons showBorderBottom={false} />);
 
-            expect(wrapper.find(ActionBarConnected).prop('extraContainerClasses')).toContain('mod-border-bottom');
+            expect(wrapper.find(ActionBarConnected).prop('extraContainerClasses')).not.toContain('mod-border-bottom');

```

Comment:
I think we could have both tests, one that tests that when `showBorderBottom` is set to false, the class `mod-border-bottom` is not there, and the other one for the opposite case.

Reply:
yeah sure I'll add it